### ISSUE TITLE
About the evaluation

### ISSUE THREAD

**Issue by L-J-Yuan** [ac-rad/MVTrans#8]
What's the difference between training and evaluation other than the training data set path? Why do gradient updates occur during evaluation ? Should I retrain the evaluation with a pre-trained model ? And Where does the evaluation index in the paper come from？
